### PR TITLE
Create a FAQ document for Sealed Secrets public documentation

### PR DESCRIPTION
**Pull request by agarcia-oss**
**Description of the change**

The Sealed Secrets public site needs a new page for frequently asked questions. This page will replace the [current entry](https://github.com/bitnami-labs/sealed-secrets#faq) in the README file.

Unrelated, the PR also includes a TOC section in the `cryptography` document

### CODE REVIEW

File: site/content/docs/latest/background/cryptography.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # Cryptography details of Sealed Secrets
 
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```
Comment (alemorcuq) [inline site/content/docs/latest/background/cryptography.md:3]: How is this file related to this PR?

File: site/content/docs/latest/reference/faq.md [hunk 2]
```diff
@@ -1,3 +1,108 @@
...
+## Will you still be able to decrypt if you no longer have access to your cluster?
+
+No, the private keys are only stored in the Secret managed by the controller (unless you have some other backup of your k8s objects). There are no backdoors - without that private key used to encrypt a given SealedSecrets, you can't decrypt it. If you can't get to the Secrets with the encryption keys, and you also can't get to the decrypted versions of your Secrets live in the cluster, then you will need to regenerate new passwords for everything, seal them again with a new sealing key, etc.
+
+## How can I do a backup of my SealedSecrets?
+
+If you do want to make a backup of the encryption private keys, it's easy to do from an account with suitable access:
+
+```console
+$ kubectl get secret -n kube-system -l sealedsecrets.bitnami.com/sealed-secrets-key -o yaml >main.key
```
Comment (alemorcuq) [inline site/content/docs/latest/reference/faq.md:27]: Are we choosing to use leading `$` for commands or not? I see uses of both alternatives.

File: site/content/docs/latest/background/cryptography.md (hunk 1 above)
Comment (agarcia-oss) [reply site/content/docs/latest/background/cryptography.md:3]: I'm using the [doctoc](https://github.com/bitnami-labs/sealed-secrets/tree/main/docs/developer#prerequisites) generation format for the document TOC. Is that not correct?

File: site/content/docs/latest/reference/faq.md (hunk 2 above)
Comment (agarcia-oss) [reply site/content/docs/latest/reference/faq.md:27]: I think the non-written rule is to use

File: site/content/docs/latest/background/cryptography.md (hunk 1 above)
Comment (alemorcuq) [reply site/content/docs/latest/background/cryptography.md:3]: Just doublechecking as the PR only mentions a FAQ and not a TOC for the Cryptography document.

File: site/content/docs/latest/reference/faq.md (hunk 2 above)
Comment (alemorcuq) [reply site/content/docs/latest/reference/faq.md:27]: Then we should change all the other instances in these two files to use, I think.

Comment (alemorcuq) [reply site/content/docs/latest/reference/faq.md:27]: We don't seem to be following that convention in these files, we should fix that.

File: site/content/docs/latest/background/cryptography.md (hunk 1 above)
Comment (agarcia-oss) [reply site/content/docs/latest/background/cryptography.md:3]: Ahh true, sorry, that was not related, fixing the PR description

File: site/content/docs/latest/reference/faq.md (hunk 2 above)
Comment (alemorcuq) [reply site/content/docs/latest/reference/faq.md:27]: I guess this one should be shell as it has no outputs

File: site/content/docs/latest/background/cryptography.md [hunk 3]
```diff
@@ -43,7 +74,7 @@ The public key (in the form of a self-signed certificate if it was generated by
 > Note that it is possible to use your own X509 certificate with the command bellow:
 
 ```console
```
Comment (alemorcuq) [inline site/content/docs/latest/background/cryptography.md:76]: I imagine this is `shell` too
```suggestion
```shell
```

Comment (agarcia-oss) [reply site/content/docs/latest/background/cryptography.md:76]: I reviewed the code examples in the page, including the command output where it provides extra information to users.

**Review by alvneiayu — CHANGES_REQUESTED** [review]

**Review by alemorcuq — APPROVED** [review]

**Review by alvneiayu — APPROVED** [review]
LGTM
